docs/Concepts/Weak-Subjectivity.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Describe weak subjectivity
+---
+
+# Weak subjectivity
+
+The weak subjectivity period refers to how far behind the chain head a node can be before knowing
+that 1/3 of validators have exited.
+
+For example, if 1/3 of validators withdraw their stake and continue signing blocks and
+attestations, they can form a chain which conflicts with the finalised state. If your node is far
+enough behind the chain head not to be aware that they've withdrawn their funds, these validators
+can act dishonestly and continue feeding you blocks to lead you down the wrong chain.
+
+!!! note
+
+    If a node is aware that a validator has withdrawn its funds, the node will reject the
+    validator's attestations.
+
+Teku allows you to [supply a weak subjectivity checkpoint], which is a point in the network from which
+a node can securely update their view of the current state.
```

Comment:
that can be downloaded by rest api: `/teku/v1/beacon/states/finalized`

Reply:
There's two different things which may be confusing here.  You can specify a checkpoint as `<blockRoot>:<epoch>` and Teku will sync from genesis and check that when it reaches that epoch that the block root matches.  This is the information you get from BeaconScan.

The other is a a checkpoint state that Teku can start syncing from which you can get from the endpoint Paul mentioned. Starting from a checkpoint state is an even better way to avoid risks with the weak subjectivity period because there's no way to be lead down a wrong chain (whereas when syncing from genesis you may follow the wrong chain but with a checkpoint specified it will be detected and Teku will stop).